upsertGitHubTag/deployment/index.js

```diff
@@ -267,7 +267,9 @@ function logPayloadToS3(body, deliveryId) {
   // If bucket name is not null (had to put this for the integration test)
   if (process.env.BUCKET_NAME) {
     const uploadDate = new Date();
-    const bucketPath = `${uploadDate.getFullYear()}-${uploadDate.getMonth()}-${uploadDate.getDay()}/${deliveryId}`; //formats path to YYYY-MM-DD/deliveryid
+    const bucketPath = `${uploadDate.getFullYear()}
+          -${uploadDate.getMonth() + 1}
+          -${uploadDate.getDate()}/${deliveryId}`; //formats path to YYYY-MM-DD/deliveryid
```

Comment:
Worth double-checking that the date is 0 or 1 indexed?

Reply:
The date seems to be 1 indexed from this info: https://www.freecodecamp.org/news/javascript-get-current-date-todays-date-in-js/